apps/web/pages/event-types/[type].tsx

```diff
@@ -1312,6 +1338,109 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
+                              </div>
+                              <div className="text-sm ltr:ml-3 rtl:mr-3">
+                                <p className="text-neutral-900">{t("recurring_event_description")}</p>
+                              </div>
+                            </div>
+                            <Collapsible
+                              open={recurringEventDefined}
+                              onOpenChange={() => setRecurringEventDefined(!recurringEventDefined)}>
+                              <CollapsibleContent className="mt-4 text-sm">
+                                <div className="flex items-center">
+                                  <p className="mr-2 text-neutral-900">{t("repeats_every")}</p>
+                                  <input
+                                    type="number"
+                                    min="1"
+                                    max="20"
+                                    className="block w-16 rounded-sm border-gray-300 shadow-sm [appearance:textfield] ltr:mr-2 rtl:ml-2 sm:text-sm"
+                                    defaultValue={
+                                      (eventType.recurringEvent && eventType.recurringEvent.interval) || 1
+                                    }
+                                    onChange={(event) => {
+                                      setRecurringEventInterval(parseInt(event?.target.value));
+                                      eventType.recurringEvent.interval = parseInt(event?.target.value);
+                                      formMethods.setValue("recurringEvent", eventType.recurringEvent);
+                                    }}
+                                  />
+                                  <Select
+                                    options={recurringEventFreqOptions}
+                                    value={recurringEventFreqOptions[recurringEventFrequency]}
+                                    isSearchable={false}
+                                    className="w-18 block min-w-0 rounded-sm sm:text-sm"
+                                    onChange={(e) => {
+                                      if (e?.value) {
+                                        setRecurringEventFrequency(parseInt(e?.value));
+                                        eventType.recurringEvent.freq = parseInt(e?.value);
+                                        formMethods.setValue("recurringEvent", eventType.recurringEvent);
+                                      }
+                                    }}
+                                  />
+                                </div>
+                                <div className="mt-4 flex items-center">
+                                  <p className="mr-2 text-neutral-900">{t("max")}</p>
+                                  <input
+                                    type="number"
+                                    min="1"
+                                    max="20"
```

Comment:
Same hardcoded value for `count` which is the Y in `Max Y weeks`, the  number of occurrences.